modules/charon-core/src/main/java/org/wso2/charon3/core/protocol/endpoints/GroupResourceManager.java

```diff
@@ -932,14 +933,31 @@ private void processValueAttributeOfOperation(SCIMResourceTypeSchema schema, Pat
     }
 
     private List<Map<String, String>> transformMembersAttributeToMap(MultiValuedAttribute multiValuedMembersAttribute)
-            throws CharonException {
+            throws CharonException, BadRequestException {
 
         List<Map<String, String>> memberList = new ArrayList<>();
         List<Attribute> subValuesList = multiValuedMembersAttribute.getAttributeValues();
         for (Attribute subValue : subValuesList) {
             ComplexAttribute complexAttribute = (ComplexAttribute) subValue;
             Map<String, Attribute> subAttributesList = complexAttribute.getSubAttributesList();
```

Comment:
Is there can be any case where `complexAttribute.getSubAttributesList()` can return null?

Reply:
No , it is initialized in `modules/charon-core/src/main/java/org/wso2/charon3/core/attributes/ComplexAttribute.java` and not being modified to `null` from anywhere.